Text updates for website feedback form

rfultz opened the pull request:
## Summary

- Resolves #3592 
Updated language for the feedback box and the thank you message afterward

## Impacted areas of the application

Could impact every page but the only changes were inside the feedback box and its 'thanks' message

## Screenshots

![image](https://user-images.githubusercontent.com/26720877/84043127-ea922180-a973-11ea-8f5d-66de30ca3617.png)


## Related PRs

None

## How to test

Wait for dev, review the screenshot, or
- pull the branch
- `npm i`
- `npm run build`
- `./manage.py runserver`
- check [any page](http://127.0.0.1) with the feedback box
- review the content changes
- submit
- review the thanks message
- approve

If you're the last approval
- unlabel and merge PR
- delete branch

____

JonellaCulmer commented:
@dorothyyeager

> If there is a room, let's put a blank line between "tell us how we can improve..." and "This feedback will be" (unless that's meant to be one paragraph, in which case, let's backspace on "This feedback.")

I think we can just remove the line break instead, what do you think?

> "What were you trying to do" seems like it has funny spacing between the lines. Also the other form parts have italicized titles but this one doesn't.

Good catch on the line height for "What were you trying to do..." @rfultz Can we drop that down to 1.6 rem? And possibly add some spacing below the "General feedback" label and the input box?

rfultz commented:
@dorothyyeager @JonellaCulmer I added the hyphen, fixed an open/close mismatch (`<span></em>`) and tweaked the styles. Whatcha think?

![image](https://user-images.githubusercontent.com/26720877/84168212-20530b00-aa45-11ea-8ddf-76c51a0044ba.png)

dorothyyeager commented:
> @dorothyyeager @JonellaCulmer I added the hyphen, fixed an open/close mismatch (`<span></em>`) and tweaked the styles. Whatcha think?
> 
> ![image](https://user-images.githubusercontent.com/26720877/84168212-20530b00-aa45-11ea-8ddf-76c51a0044ba.png)

@rfultz Looks good. I like Jonella's idea of removing the line break between "using this form" and "This feedback."

rfultz commented:
@JonellaCulmer @dorothyyeager here's the latest: (it's _really_ long)

![image](https://user-images.githubusercontent.com/26720877/84178950-3f589980-aa53-11ea-9ddf-b717970038ab.png)
